src/javascript/app/pages/cashier/payment_agent_list.js

```diff
@@ -75,9 +91,9 @@ const PaymentAgentList = (() => {
                     .replace(/%summary/g, agent.summary)
                     .replace(/%deposit_commission/g, agent.deposit_commission)
                     .replace(/%withdrawal_commission/g, agent.withdrawal_commission)
-                    .replace(/%url/g, agent.url)
+                    .replace(/%url/g, urls)
```

Comment:
```suggestion
                    .replace(/%url/g, agent.urls? urls: agent.url)
```
For backward compatibility, it has to fall back to the old values, hasn't it?

Reply:
do we still need to use agent.url? i thought it will be deprecated soon.

Reply:
Yes they will be removed; but we are asked to write a backward compatible code.